### ISSUE TITLE
Collada file not getting imported in Blender, Working in ThreeJs Collada Loader

### ISSUE THREAD

**Issue by sonuhawk** [gkjohnson/collada-exporter-js#18]
**Steps** 

**1) Created Collada File using exporter in Three JS** 

```js
var geometry1 = new THREE.DodecahedronGeometry(2,1);
var material1 = new THREE.MeshLambertMaterial({color:0xFFFFFF, wireframe:true});
var mesh1 = new THREE.Mesh(geometry1,material1);

var geometry2 = new THREE.SphereBufferGeometry( 2, 2, 2 );
var material2 = new THREE.MeshPhongMaterial({color:0xFF8FF, wireframe:false});var mesh2 = new THREE.Mesh( geometry2, material2 );

camera.position.z = 3;

 group = new THREE.Object3D();//create an empty container
 group.add( mesh2, mesh1 );//add a mesh with geometry to it
 Scene.add( group );//when done, add the group to the scene
        

  var loader = new THREE.ColladaLoader();
  var exp = new THREE.ColladaExporter();

  var { data } = exp.parse(group);
  console.log('Format', data);
```

**2) Imported File in Three JS collada Loader ( Working Fine )**

 
**3) Tried Importing File in Blender , Not getting imported - No errors , nothing getting displayed after importing in blender.**

[elf.zip](https://github.com/gkjohnson/collada-exporter-js/files/2197147/elf.zip)

**Comment by gkjohnson** [comment]
Hey!

It looks like you might be using an old version of the exporter. Looking at your file, it looks like the `triangles` and `vertices` tags are exported incorrectly, which I fixed in PR #7:

```xml
        <vertices id="Mesh1-vertices">
          <input semantic="POSITION" source="#Mesh1-position" />
          <input semantic="NORMAL" source="#Mesh1-normal" offset="0" />
          <input semantic="TEXCOORD" source="#Mesh1-texcoord" offset="0" />
        </vertices>
        <triangles material="MESH_MATERIAL_0" count="6">
          <input semantic="VERTEX" source="#Mesh1-vertices" offset="0" />
          <p>0 4 5 1 5 6 2 6 7 5 4 9 6 5 10 7 6 11</p>
        </triangles>
```
Should be
```xml
        <vertices id="Mesh1-vertices">
          <input semantic="POSITION" source="#Mesh1-position" />
        </vertices>
        <triangles material="MESH_MATERIAL_0" count="6">
          <input semantic="VERTEX" source="#Mesh1-vertices" offset="0" />
          <input semantic="NORMAL" source="#Mesh1-normal" offset="0" />
          <input semantic="TEXCOORD" source="#Mesh1-texcoord" offset="0" />
          <p>0 4 5 1 5 6 2 6 7 5 4 9 6 5 10 7 6 11</p>
        </triangles>
```

Can you try the newest version and let me know how it goes? Keep in mind that there's another (minor) bug that was fixed in PR https://github.com/gkjohnson/collada-exporter-js/pull/16 that isn't up on NPM, yet.

**Comment by sonuhawk** [comment]
Hey - I tried your updated file, Export is coming good :) Thank you
